app/src/tx_reviewer.rs

```diff
@@ -314,10 +314,10 @@ impl TxReviewer {
                 let inputs_count = current_input_index - last_input_index;
                 let review_message_from_index = self.index;
                 let review_message_to_index = if inputs_count == 1 {
-                    self.write_index_with_prefix(last_input_index, b"Review Input #")?
+                    self.write_index_with_prefix(last_input_index, b"Input #")?
                 } else {
-                    let prefix = b"Review Inputs #";
-                    let mut bytes = [0u8; 25];
+                    let prefix = b"Inputs #";
```

Comment:
On nanos, if the string is too long, it causes the latter part to be truncated and not displayed, so I removed the `Review` prefix.